### PR TITLE
Correction for the name of offline safe algorithm COptiDICE and bug of load_state_dict in normalizer

### PR DESCRIPTION
**Pull request by Komorebi-wd**
# Description
The algorithm name for COptiDICE was written incorrectly.

If I use the code part /* return super().load_state_dict(state_dict, strict) */ , the following error will generate
image
something about this error 
image

Describe your changes in detail.
change COptDICE to COptiDICE
change return super().load_state_dict(state_dict, strict, assign) to return super().load_state_dict(state_dict, strict)
## Motivation and Context

Why is this change required? What problem does it solve?
If it fixes an open issue, please link to the issue here.
You can use the syntax `close #15213` if this solves the issue #15213

- [ ] I have raised an issue to propose this change ([required](https://github.com/PKU-Alignment/omnisafe/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [ ] I have read the [CONTRIBUTION](https://github.com/PKU-Alignment/omnisafe/blob/HEAD/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.
- [ ] I have reformatted the code using `make format`. (**required**)
- [ ] I have checked the code using `make lint`. (**required**)
- [ ] I have ensured `make test` pass. (**required**)

### CODE REVIEW

**Comment by Gaiejj** [comment]
I think this change is correct and necessary. I agree with your opinion.

**Review by Gaiejj — DISMISSED** [review]

**Comment by Komorebi-wd** [comment]
my torch version is 2.0.1
image

**Comment by Gaiejj** [comment]
I think updating torch to >=2.1.0 can resolve this arguments issue:
image

**Comment by Komorebi-wd** [comment]
OK，thank you for your explanation.

**Comment by Komorebi-wd** [comment]
In benchmark/readme, there make be something wrong 
image
is it CRR , but not CCR?
